conf/documentation.conf

```diff
@@ -868,6 +868,20 @@ description=<<EOT
 The max_connect_errors MySQL configuration attribute. Only change if you know what you are doing. Will only affect a locally running MySQL server.
 EOT
 
+[database_advanced.masterslave]
+type=toggle
+options=ON|OFF
+description=<<EOT
+The masterslave configuration to enable or disable the master/slave configuration.
+EOT
+
+[database_advanced.other_members]
+type=text
+description=<<EOT
+Comma delimited IPv4 address of other member mysql members - note that this is only to sync
+the database.
```

Comment:
We could also clarify that this parameter is to be used only with the read-only mode or a specialized configuration and not with a standard cluster

Reply:
yeah it's candy, it's like when you want to add other mariadb members in the cluster but these members are not some packetfence servers.